51-android.rules

```diff
@@ -362,7 +362,7 @@ ATTR{idVendor}!="2b0e", GOTO="not_letv"
 #   LEX720 LeEco Pro3 6GB (610c=normal,610d=debug, 610b=camera)
 ATTR{idProduct}=="610d", GOTO="adbfast"
 ENV{adb_user}="yes"
-GOTO="android_usb_rules_end"
+GOTO="android_usb_rule_match"
 LABEL="not_letv"
```

Comment:
my long distance view now, would be to avoid global wrappers and get more specific.
The global wrappers can interfere with downstream rules, like 52-...53-....,etc.
A good example, Some of the larger manufacturers like Samsung, have other devices beyond just android phones, so if 51-android.rules targets just android devices, that leaves other rules to take care of other stuff like joysticks, cameras, scanners, and much more.
Getting more specific info means we have to do some homework to clean this up a bit, example idVendor-2b0e
http://www.linux-usb.org/usb.ids
https://www.google.com/search?client=firefox-b-e&q=idVendor+2b0e

Reply:
Yes, gathering more device-specific information is what we should strive for and sooner or later a breaking change that gets rid of all vendor-wide `ENV{*}="yes"` catch-all rules would be in order. However, as I understood, #266 didn't intend to do that just yet and in that light this line should wait a little more.

Reply:
big patch - missed it :-)